Lib/distutils/util.py

```diff
@@ -90,8 +92,25 @@ def get_platform ():
 
     return "%s-%s-%s" % (osname, release, machine)
 
-# get_platform ()
+# get_host_platform ()
+
+def get_platform ():
+    TARGET_TO_PLAT = {
+        'x86' : 'win32',
+        'x64' : 'win-amd64',
+        'arm' : 'win-arm32',
+    }
+
+    targetPlatformFromEnvironment = os.environ.get('VSCMD_ARG_TGT_ARCH')
 
+    if targetPlatformFromEnvironment != None and targetPlatformFromEnvironment in TARGET_TO_PLAT:
+        targetPlatform = TARGET_TO_PLAT[targetPlatformFromEnvironment]
+    else:
+        targetPlatform = get_host_platform()
+
+    return targetPlatform
```

Comment:
These six lines can simplify down to:

```python
return TARGET_TO_PLAT.get(os.environ.get('VSCMD_ARG_TGT_ARCH')) or get_host_platform()
```

Reply:
And we should put the environment check inside an `if os.name == 'nt'` block (matching `get_platform()`s check), or else just return `get_host_platform()`.